scripts/utils.py

```diff
@@ -3,7 +3,7 @@
 
 def write_json_to_file(data, output_name):
   with open(output_name, 'w') as outfile:
-    json.dump(data, outfile, indent=4)
+    json.dump(data, outfile, indent=4, sort_keys=True)
```

Comment:
I thought it would be useful to sort the packages.. this was a user request in the past IIRC.